modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: REFERENCE
+[id="nw-aws-load-balancer-operator-considerations_{context}"]
+= AWS Load Balancer Operator considerations
+
+Review the following limitations before installing and using the AWS Load Balancer Operator (ALBO).
+
+* After installing the ALBO, by default, controller command-line flags such as link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/deploy/configurations/#disable-ingress-class-annotation[`--disable-ingress-class-annotation`] and link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/deploy/configurations/#disable-ingress-group-name-annotation[`--disable-ingress-group-name-annotation`] are added to the AWS Load Balancer Controller. Therefore, the ALBO does not allow using the `kubernetes.io/ingress.class` and `kubernetes.io/ingress.class` annotations in the `AWSLoadBalancerController` custom resource (CR).
+
+* The IP target mode only works on AWS Elastic Kubernetes Service (EKS). The ALBO disables the link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/how-it-works/#ip-mode[IP target mode] for the AWS Load Balancer Controller and hence, the AWS Load Balancer Controller does not use the link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/deploy/pod_readiness_gate/[pod readiness gate].
```

Comment:
Let's put this line first because it's more important.

I would like to abstain from linking the upstream documentation. We can just say `IP traffic mode` (`traffic` is more commonly used than `target`) and `pod readiness gate`. The reader should understand the context. But up to you to clarify the point about the linking to the upstream docs.